src/components/YourCompany/CompanyName/CorrectNameRequest.vue

```diff
@@ -176,9 +183,10 @@ export default class CorrectNameRequest extends Mixins(CommonMixin, NameRequestM
   }
 
   /** Inform parent when form is valid and ready for submission. */
-  @Watch('valid')
-  @Watch('entityPhone')
-  @Watch('entityEmail')
+  @Watch('formValid')
+  @Watch('nameRequestNumber')
```

Comment:
I added a watch for name request number here. It seems to make sense. I can't think of a reason why we would not watch for changes to the NR number... ?